apps/marketplace/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+
+
+@NgModule({
+  declarations: [],
+  imports: [
+    CommonModule
```

Comment:
here you would add the `RouterModule.forRoot([{path: '', component: AppComponent}])` and so on. If you want to have lazy loading, dont use the component key, use `loadChildren`

Reply:
or in this case your LandingComponent.

Reply:
Fixed the issue by defining the correct module.. used the right CLI.